scripts/sitemap.py

```
@@ -0,0 +1,110 @@
+
+    def generate_json(self) -> str:
+        """生成 JSON 格式（供前端使用）"""
+        return json.dumps({
+            "generated_at": datetime.now().isoformat(),
+            "total_urls": len(self.urls),
+            "urls": self.urls
+        }, indent=2)
+
+
+def main():
+    # 示例：生成站点地图
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)